setup.cfg

```diff
@@ -26,3 +26,16 @@ exclude =
     site-static,
     static,
     .git
+
+[isort]
+atomic=true
+lines_after_imports=2
+lines_between_types=1
+multi_line_output=4
+combine_as_imports = true
+default_section = THIRDPARTY
+include_trailing_comma = false
+known_django = django,rest_framework
```

Comment:
On the one hand, `rest_framework_nested` and `rest_framework_jwt` should probably be in there? It feels weird to see them *before* DRF. On the other, where do we stop? There are a bunch of django-related packages for which we can say the same thing...